utils.js

```diff
@@ -398,8 +398,8 @@ function isDomElement(thing) {
  * execute any of its inline scripts.
  */
 function staticDom(html) {
-    const {jsdom} = require('jsdom');
-    return jsdom(html, {features: {ProcessExternalResources: false}});
+    var parser = new DOMParser();
```

Comment:
I was hoping there would be a flag in DOMParser or something. Failing all else, we can hand-remove the script tags for now, but we'll eventually want a way to parse untrusted pages server-side and still measure distances.